packages/flutter/lib/src/material/icon.dart

```diff
@@ -22,6 +24,14 @@ const Map<IconSize, int> _kIconSize = const <IconSize, int>{
   IconSize.s48: 48,
 };
 
+SplayTreeMap _kIconDensities = new SplayTreeMap.from({
+  1.0 : 'mdpi',
+  1.5 : 'hdpi',
+  2.0 : 'xhdpi',
+  3.0 : 'xxhdpi',
+  4.0 : 'xxxhdpi'
+});
```

Comment:
Rather than having this customized list, I think we should change how we store material design icons in the FLX to match the pattern that we think folks should use for other assets.